tpgtools/samples/assuredworkloads/workload/workload_post_destory.go

```diff
@@ -0,0 +1,78 @@
+// deleteAssuredWorkloadProvisionedResources deletes the resources provisioned by
```

Comment:
filename should be `workload_post_destroy`?

Reply:
changed